extension/snippets/xcomponent.json

```diff
@@ -8,6 +8,13 @@
 			"</StateData>"
 		]
 	},
+	"Create a triggered method": {
+		"prefix": "xctriggeredmethod",
+		"description": "Create a triggered method",
+		"body": [
+			"<TriggeredMethodData IsNodeInitializer=\"${1|false,true|}\" IsSelected=\"${2|true,false|}\" TransitionName=\"${3}\" TriggeringEvent=\"${4}\" />"
```

Comment:
missing: IsExternal ;)

Reply:
Brand new property.
I will add it a next PR